src/test/java/edu/hm/hafner/analysis/parser/GrypeParserWoDescriptionTest.java

```diff
@@ -0,0 +1,46 @@
+package edu.hm.hafner.analysis.parser;
+
+import edu.hm.hafner.analysis.AbstractParserTest;
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+import edu.hm.hafner.analysis.assertions.SoftAssertions;
+
+import static j2html.TagCreator.a;
+import static j2html.TagCreator.p;
+
+class GrypeParserWoDescriptionTest extends AbstractParserTest {
+    protected GrypeParserWoDescriptionTest() {
+        super("grype-report-wo-description.json");
+    }
+
+    @Override
+    protected void assertThatIssuesArePresent(final Report report, final SoftAssertions softly) {
```

Comment:
Can you please move this code to a new test method in `GrypeParserTest`? YOu can use `readReport` to obtain the warnings, see some other tests.

Reply:
I'm sorry, but I can't find any method named `readReport`. Can you please provide some more details on where this method should exist?

Reply:
I should have looked into the IDE, the method is called `parse` 😄.

Here are some examples: https://github.com/jenkinsci/analysis-model/blob/main/src/test/java/edu/hm/hafner/analysis/parser/AntJavacParserTest.java

Reply:
Thanks! I've updated the test using the `parse` method. I hope it looks fine now.